lib/adapters/REST/endpoints/organization.ts

```diff
@@ -1,14 +1,17 @@
 import type { AxiosInstance } from 'contentful-sdk-core'
-import { CollectionProp } from '../../../common-types'
+import { CollectionProp, GetOrganizationParams } from '../../../common-types'
 import { OrganizationProp } from '../../../entities/organization'
-import { GetOrganizationParams } from '../../../plain/common-types'
+import { RestEndpoint } from '../types'
 import * as raw from './raw'
 
-export const getAll = (http: AxiosInstance) => {
+export const getAll: RestEndpoint<'Organization', 'getAll'> = (http: AxiosInstance) => {
```

Comment:
I would make it `getMany` too, for consistency™️. We can still export also `getAll` for backwards compatibility and get rid of it on the next major